soh/soh/Enhancements/randomizer/logic.cpp

```diff
@@ -816,8 +816,7 @@ namespace Rando {
     }
 
     bool Logic::CanBreakMudWalls() {
-        //RANDOTODO blue fire tricks
-        return BlastOrSmash();
+        return BlastOrSmash() || (ctx->GetTrickOption(RT_BLUE_FIRE_MUD_WALLS) && BlueFire());
```

Comment:
can't wait for this to require going between ice cavern & goron city with one bottle in order to reach Medigoron